Gem/Code/Source/Components/NetworkRigidBodyComponent.cpp

```diff
@@ -75,7 +75,21 @@ namespace MultiplayerSample
         AzPhysics::RigidBody* rigidBody = m_physicsRigidBodyComponent->GetRigidBody();
         rigidBody->SetFrameId(frameId);
 
-        const AZ::Transform& rewoundTransform = m_transform.Get();
+        AZ::Transform rewoundTransform;
+        const AZ::Transform& targetTransform = m_transform.Get();
+        const float blendFactor = Multiplayer::GetNetworkTime()->GetHostBlendFactor();
+        if (blendFactor < 1.f)
+        {
+            // If a blend factor was supplied, interpolate the transform appropriately
```

Comment:
If this comes up elsewhere, it would be good to create a method for this transform lerping.